app/routers/channels.py

```diff
@@ -0,0 +1,93 @@
+from fastapi import APIRouter
+import schemas
+
+
+channels_router = APIRouter(
+    prefix = "/channels",
+    tags = ["channels"]
+)
+
+
+@channels_router.get("/")
+async def get_list():
+    return {'status':'OK'}
+
+
+@channels_router.post("/")
+async def create_channel(info: schemas.Channel):
+    return {'status':'OK'}
+
+
+@channels_router.get("/{channel_id}")
+async def get_info_channel(info: schemas.Channel):
+    return {'channel_id':'channel_id'}
+
+
+@channels_router.patch("/{channel_id}")
+async def edit_info_channel(info: schemas.Channel):

```

Comment:
Опять же, объявленный в пути параметр не передан в функцию.
Далее по коду это также встречается, отдельно отмечать не буду.

Reply:
К тому же схема Channel не подходит для описания параметров, требуемых на вход. Ты путаешь входные и возвращаемые параметры. В схеме, nребующейся на вход, должны присутствовать только 2 параметра: title и description.
Также данный эндпоинт принимает параметр photo, который передаётся как бинарный объект и его следует объявить таким образом
```suggestion
async def edit_info_channel(photo: UploadFile, info: schemas.Channel):
```
Класс UploadFile предварительно импортируется из fastapi